<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Система эксплуатационного учета "Платон"</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images/resources.qrc" >
    <normaloff>:/PICS/konstr.png</normaloff>:/PICS/konstr.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="cursor" >
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet" >
       <string notr="true" >font: 10pt "Albany AMT";</string>
      </property>
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <property name="iconSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_search" >
       <attribute name="title" >
        <string>Поиск</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_explore" >
       <attribute name="title" >
        <string>Объектное представление</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QComboBox" name="comboBox_Species" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <item>
             <property name="text" >
              <string>Все</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Объекты</string>
             </property>
             <property name="icon" >
              <iconset resource="images/resources.qrc" >
               <normaloff>:/PICS/browser.png</normaloff>:/PICS/browser.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Действия</string>
             </property>
             <property name="icon" >
              <iconset resource="images/resources.qrc" >
               <normaloff>:/PICS/action.png</normaloff>:/PICS/action.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Ресурсы</string>
             </property>
             <property name="icon" >
              <iconset resource="images/resources.qrc" >
               <normaloff>:/PICS/diamond_juliane_krug_01.png</normaloff>:/PICS/diamond_juliane_krug_01.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>НСИ</string>
             </property>
             <property name="icon" >
              <iconset resource="images/resources.qrc" >
               <normaloff>:/PICS/contents.png</normaloff>:/PICS/contents.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_e" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QEidosTreeWidget" name="EidosTreeWidget" >
           <property name="styleSheet" >
            <string notr="true" >font: 10pt "Arial";</string>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <column>
            <property name="text" >
             <string>Наименование</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>Parent_ID</string>
            </property>
           </column>
          </widget>
          <widget class="QSplitter" name="splitter_hp" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTableView" name="tableViewHypotesis" >
            <property name="font" >
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet" >
             <string notr="true" >font: 10pt "Arial";</string>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::AllEditTriggers</set>
            </property>
            <property name="tabKeyNavigation" >
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors" >
             <bool>true</bool>
            </property>
            <property name="selectionMode" >
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior" >
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
           </widget>
           <widget class="QTableView" name="tableViewPragma" />
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu" >
    <property name="title" >
     <string>Файл</string>
    </property>
    <addaction name="action_quit" />
   </widget>
   <widget class="QMenu" name="menu_2" >
    <property name="title" >
     <string>Запись</string>
    </property>
    <addaction name="action_add" />
    <addaction name="action_del" />
    <addaction name="action_refresh" />
    <addaction name="action_links" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menu_3" >
    <property name="title" >
     <string>Окно</string>
    </property>
    <addaction name="action_AddFilter" />
    <addaction name="action_RemoveFilter" />
   </widget>
   <addaction name="menu" />
   <addaction name="menu_2" />
   <addaction name="menu_3" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="layoutDirection" >
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
   <addaction name="action_quit" />
   <addaction name="separator" />
   <addaction name="action_edit" />
   <addaction name="action_add" />
   <addaction name="action_del" />
   <addaction name="separator" />
   <addaction name="action_refresh" />
   <addaction name="action_links" />
   <addaction name="action_AddFilter" />
   <addaction name="action_RemoveFilter" />
  </widget>
  <action name="action_quit" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/exit.png</normaloff>:/PICS/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Выход</string>
   </property>
  </action>
  <action name="action_edit" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/edit.png</normaloff>:/PICS/edit.png</iconset>
   </property>
   <property name="text" >
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_add" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/edit_add.png</normaloff>:/PICS/edit_add.png</iconset>
   </property>
   <property name="text" >
    <string>Добавить</string>
   </property>
  </action>
  <action name="action_del" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/trashcan_full.png</normaloff>:/PICS/trashcan_full.png</iconset>
   </property>
   <property name="text" >
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_refresh" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/reload.png</normaloff>:/PICS/reload.png</iconset>
   </property>
   <property name="text" >
    <string>Обновить</string>
   </property>
  </action>
  <action name="action_links" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/chains.png</normaloff>:/PICS/chains.png</iconset>
   </property>
   <property name="text" >
    <string>Ссылки в базе</string>
   </property>
  </action>
  <action name="action_AddFilter" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/filter2.png</normaloff>:/PICS/filter2.png</iconset>
   </property>
   <property name="text" >
    <string>Добавить фильтр</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_RemoveFilter" >
   <property name="icon" >
    <iconset resource="images/resources.qrc" >
     <normaloff>:/PICS/filteroff.png</normaloff>:/PICS/filteroff.png</iconset>
   </property>
   <property name="text" >
    <string>Убрать условия фильтрации</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEidosTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qeidostreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images/resources.qrc" />
 </resources>
 <connections/>
</ui>
